color/p5js/color-contrast/index.md

```diff
@@ -0,0 +1,10 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Contrast
+tags: P5.js, "Law of Closure", "Law of Connected-Elements", "Law of Contrast", "Law of Figure-Ground", "Law of Regularity", "Law of Similarity", "Law of Simplicity", "Law of Symmetry", Interaction, Contrast
+permalink: /contrast/Contrast/
```

Comment:
the permalink should contain the category and the new name of your sketch. also it should be lowercase and have the name of the folder. 

     /color/color-contrast/